Doc/library/asyncio-eventloop.rst

```diff
@@ -1090,6 +1089,22 @@ Handle
 
       .. versionadded:: 3.7
 
+.. class:: TimerHandle
+
+   A callback wrapper object returned by :func:`AbstractEventLoop.call_later`,
+   and :func:`AbstractEventLoop.call_at`.
+
+   The class is inherited from :class:`Handle`.
+
+   .. method:: when()
+
+      Return a scheduled callback time as :class:`float` seconds.
+
+      The time is an absolute timestamp, using the same time
+      reference as :meth:`AbstractEventLoop.time`.
+
+      .. versionadded:: 3.7
```

Comment:
`..versionadded: 3.8` ?

Reply:
Let's wait for @ned-deily decision

Reply:
@asvetlov decision taken by @ned-deily ;-) welcome to 3.7